rem/remApp.swift

```diff
@@ -360,10 +367,11 @@ class AppDelegate: NSObject, NSApplicationDelegate {
             ffmpegProcess.executableURL = URL(fileURLWithPath: ffmpegPath)
             ffmpegProcess.arguments = [
                 "-f", "image2pipe",
+                "-vcodec", "png",
                 "-i", "-",
                 "-color_trc", "iec61966_2_1", // Set transfer characteristics for sRGB (approximates 2.2 gamma)
                 "-c:v", "h264_videotoolbox",
-                "-crf", "25",
+                "-q:v", "25",
```

Comment:
someone who is better at ffmpeg... take a look at this and you know, make it work better 😅